src/core_plugins/kibana/public/home/components/recently_accessed.js

```diff
@@ -0,0 +1,188 @@
+import './recently_accessed.less';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiPanel,
+  EuiLink,
+  EuiText,
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiIcon,
+} from '@elastic/eui';
+
+export const NUM_LONG_LINKS = 5;
+
+export class RecentlyAccessed extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isPopoverOpen: false,
+    };
+  }
+
+  onButtonClick() {
+    this.setState({
+      isPopoverOpen: !this.state.isPopoverOpen,
+    });
+  }
+
+  closePopover() {
+    this.setState({
+      isPopoverOpen: false,
+    });
+  }
+
+  renderDropdown = () => {
+    const dropdownLinks = [];
+    for (let i = NUM_LONG_LINKS; i < this.props.recentlyAccessed.length; i++) {
+      dropdownLinks.push(
+        (
+          <li
+            style={{ marginBottom: 8 }}
+            key={this.props.recentlyAccessed[i].link}
+          >
+            <EuiLink
+              className="dropwdownLink"
+              href={this.props.recentlyAccessed[i].link}
+            >
+              {this.props.recentlyAccessed[i].label}
+            </EuiLink>
+          </li>
+        )
+      );
+    }
+
+    const openPopoverComponent = (
+      <EuiLink onClick={this.onButtonClick.bind(this)}>
+        <EuiTextColor color="subdued">
+          {`+${dropdownLinks.length}`}
```

Comment:
Personally, I feel like this would be better spelled out, e.g. "5 more links".  The +{number} isn't quite intuitive to me. @gchaps Thoughts?

![image](https://user-images.githubusercontent.com/1238659/36004934-0e7bb102-0cea-11e8-8b3b-ef0f0ff1d972.png)

Reply:
I agree, @cjcenizal "5 more links" is a better way to convey the meaning.